src/gui/creds/flow2auth.cpp

```diff
@@ -82,6 +82,7 @@ void Flow2Auth::fetchNewToken(const TokenAction action)
     // add 'Content-Length: 0' header (see https://github.com/nextcloud/desktop/issues/1473)
     QNetworkRequest req;
     req.setHeader(QNetworkRequest::ContentLengthHeader, "0");
+    req.setRawHeader("User-Agent", Utility::friendlyUserAgentString());
```

Comment:
Technically not needed, only style-nitpicking but as headers are case-insensitive (and in the general case of HTTP/2: necessarily lowercased before transmission) I'd set is as lower-case.

Reply:
Yes they are case-insensitive that's why most upper case initials for readability (that includes me and most docs one would find). Internally Qt will call `toLower()` on all the headers anyway (qhttp2protocolhandler.cpp `build_headers` function) so it doesn't matter much in application code regarding the behavior.

That said, even though I'd be a proponent of keeping the header string that way (in case that wasn't obvious yet)... you made me realize that we don't have to use `setRawHeader` here! `setHeader` should work just fine since we have the right header enum value. I'll add a commit to switch to this.

Reply:
Done